src/modules/aws_acm/mappers/certificate.ts

```diff
@@ -79,7 +87,23 @@ export class CertificateMapper extends MapperBase<Certificate> {
     return Object.values<string>(certificateStatusEnum).includes(t);
   }
 
-  certificateMapper(e: CertificateDetail, region: string) {
+  getCertificateTags = crudBuilderFormat<ACM, 'listTagsForCertificate', Tag[] | undefined>(
+    'listTagsForCertificate',
+    CertificateArn => ({ CertificateArn }),
+    res => res?.Tags,
```

Comment:
Nit: This could be `res?.Tags ?? []` and then the type could be simpler and there's less guarding later on in the code.